src/components/forms/riddleUpsertForm/RiddleBasicInformationForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Button, Stack } from '@mui/material';
+import type { UseFormReturn } from 'react-hook-form-mui';
+import { FormContainer, TextFieldElement } from 'react-hook-form-mui';
+import type { Dispatch, SetStateAction } from 'react';
+import React from 'react';
+
+import { FileUploader } from '../common/FileUploader';
+import { AutocompleteLanguages } from '../common/AutocompleteLanguages';
+import { AutocompleteDifficulties } from '../common/AutocompleteDifficulties';
+import type { RiddleUpsertDetail } from '../../../utils/Types';
+
+type Props = {
+	formContext: UseFormReturn<RiddleUpsertDetail>;
+	setRiddleName: Dispatch<SetStateAction<string | null>>;
+	handleNext: () => void;
+	onCancel: () => void;
+};
+
+export const RiddleBasicInformationForm = ({
+	formContext,
+	setRiddleName,
+	handleNext,
+	onCancel
+}: Props) => {
+	const { control } = formContext;
+	return (
+		<FormContainer formContext={formContext}>
+			<Stack gap={2} sx={{ minWidth: { md: 500 } }}>
+				<TextFieldElement
+					name="name"
+					label="Riddle name"
+					// required
+					onChange={e => setRiddleName(e.target.value)}
+				/>
+				<TextFieldElement
+					label="Description"
+					multiline
+					name="description"
+					rows={5}
+					// required
+				/>
+				<FileUploader
+					name="image"
```

Comment:
Since everything seems to be correctly passed, I don't understand why the uploaders don't work :(